app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -180,6 +181,24 @@ def log
     @log ||= LettingsLog.new(attributes_for_log)
   end
 
+  def log_already_exists?
+    fields_for_duplicity_check = %w[
+      startdate
+      postcode_full
+      brent
+      scharge
+      pscharge
+      supcharg
+      tenancycode
+      age1
+      sex1
+      ecstat1
+      ethnic
+    ]
+
+    LettingsLog.exists?(fields_for_duplicity_check.index_with { |field| log[field] })
```

Comment:
I don't think duplicity is the word here, unless you think the uploader is doing it on purpose to test us.
Maybe duplicate/replica